common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1120,6 +1120,15 @@ def _block_matches_all(block_data):
         else:
             return []
 
+    def has_path_to_root(self, xblock, course):
```

Comment:
Added an assert `assert isinstance(xblock, BlockKey)` although we can would like to raise `ItemNotFoundError`, we can also use the following.

```
if not isinstance(xblock, BlockKey):
    raise ItemNotFoundError(xblock)
```